config/default.config.js

```diff
@@ -49,7 +49,7 @@ module.exports = {
     servePublic: true, // Tell app.js serve out its public folder (js, css, etc)
     serveStore: true, // Tell app.js serve out store public folder
 
-    logPath: './logs', // Folder for logs
+    logPath: '', // Folder for logs
```

Comment:
Let's add a comment, describing that adding that property will enable file logs but by default it is off